src/routes/controllers/project.controller.js

```diff
@@ -35,6 +36,49 @@ const createProject = async (req, res, next) => {
   }
 };
 
+const editProject = async (req, res, next) => {
+  try {
+    const { projectId } = req.params;
+    const { title } = req.body;
+    const { _id: userId } = req.user;
+
+    if (!mongoose.isValidObjectId(projectId) || title === undefined) {
+      return next(createError(400, BAD_REQUEST));
+    }
+
+    await Project.findOneAndUpdate(
+      { _id: projectId, owner: userId },
+      { title }
+    );
```

Comment:
```suggestion
   const updatedProject = await Project.findOneAndUpdate(
      { _id: projectId, owner: userId },
      { title }
   ).exec();
```
위같이 수정값을 얻어서 제대로 수정 안 됐을때 에러를 발생시키는 로직이 포함되면 좋을 것 같습니다!

Reply:
좋은 의견입니다~
```suggestion
    await Project.findOneAndUpdate(
      { _id: projectId, owner: userId },
      { title }
    ).lean().exec();
```

lean()으로 체인 걸어주면 성능이 더 나아질까요?

Reply:
네 그렇다고 알고있습니다!